distributed/pubsub.py

```diff
@@ -334,7 +353,8 @@ def _put(self, msg):
             self._buffer.append(msg)
             return
 
-        data = {"op": "pubsub-msg", "name": self.name, "msg": to_serialize(msg)}
+        # FIXME If I use to_serialize here, this breaks msgs of type dict!
+        data = {"op": "pubsub-msg", "name": self.name, "msg": msg}
```

Comment:
This broke for simple dict type messages by raising a keyerror on on the remote side. probably a bug but how to deal with the serialized objects in general would be one of my questions